docs/apis-tools/orchestration-cluster-api-rest/specifications/delete-tenant.api.mdx

```diff
@@ -47,8 +51,12 @@ Deletes an existing tenant.
       name: "tenantId",
       in: "path",
       required: true,
-      description: "The unique identifier of the tenant.",
-      schema: { type: "string" },
+      schema: {
+        type: "string",
+        format: "string<TenantId>",
```

Comment:
🔧 we should make the required pattern transparent here https://github.com/camunda/camunda-orchestration-cluster-api/blob/main/specification/rest-api.yaml#L11594